appinfo/register_command.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * ownCloud - Mail
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ *
+ * @author Christoph Wurst <christoph@winzerhof-wurst.at>
+ * @copyright Christoph Wurst 2016
+ */
+
+use OC;
+use OCA\Mail\Command\CreateAccount;
+
+$accountService = OC::$server->query('OCA\Mail\Service\AccountService');
```

Comment:
I'm aware of that and so I'm planning to leave this for 0.4 where oc7 support is dropped.